src/metadata/taglib_handler.cc

```diff
@@ -520,6 +520,13 @@ void TagLibHandler::extractASF(TagLib::IOStream* roStream, const std::shared_ptr
     }
     populateGenericTags(item, asf);
 
+    auto audioProps = asf.audioProperties();
```

Comment:
There is already handling for audioProperties in `populateGenericTags`, should be able to add it there.

Reply:
`TagLib::File` does not have the required audioProperties. The base method returns a simpler type with fewer methods.